### DISCUSSION TITLE
Argocd logs not being written inside the pod

### DISCUSSION THREAD

**Discussion by sudheer757** [argoproj/argo-cd#17470] [General]
I have a specific requirement where I need to ship argocd logs to splunk via fluentbit. I am currently using Aws Eks with fargate. The challenge I could see here in the argocd is, logs were not getting written to any files inside pod as fluentbit can expect logs to be in the files.
Can someone helps me if argocd writes logs to somewhere else other than stdout.If it is not possible atleast please suggest a way to pull argocd logs to the splunk via fluentbit. Thanks in advance

**Comment by blakepettersson** [comment]
I would suggest to look at [this guide](https://docs.aws.amazon.com/eks/latest/userguide/fargate-logging.html) (this applies for sending to Cloudwatch logs, but shouldn't be too difficult to adapt to Splunk).

**Reply by sudheer757** [reply]
We have cost concerns on our org. So instead of using cloudwatch, we will have to send logs directly to splunk via fluentbit. One big challenge here is, we are using eks with fargate. So argocd pod logs won't be available in the nodes
